### PR TITLE
Add Login as Customer inforrmation

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds information about the new Login as Customer admin button.

## Magento release version

- [x ] 2.4.1

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-company-update.html

### CODE REVIEW

File: src/customers/account-company-update.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ These role resources must be set for the [User Role]({% link system/permissions-
 | Button            | Description |
 |-------------------|-------------|
 | <span class="btn">Back</span>              | Returns to the Companies page without saving changes. |
+| <span class="btn">Login as Customer</span> | Allows logging in as an admin user to help customers with their orders. |
```
Comment (leslietilling) [inline src/customers/account-company-update.md:38]: I think this needs some clarification

```suggestion
| <span class="btn">Login as Customer</span> | Allows an Admin user to log in to the storefront as the customer and help with their orders. |
```

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by leslietilling** [comment]
Make sure this is squashed when you merge,

**Comment by dobooth** [comment]
running tests

**Pull request merged**